src/browser/application.js

```diff
@@ -1,170 +1,167 @@
-'use strict';
-
-var app = require('app');
-var ipc = require('ipc');
-var path = require('path');
-var shell = require('shell');
-var BrowserWindow = require('browser-window');
-var EventEmitter = require('events').EventEmitter;
-var _ = require('underscore-plus');
-var AppMenu = require('./appmenu');
-var AppWindow = require('./appwindow');
-
-/**
- * The application's class.
- *
- * It's the entry point into the application and maintains the global state of the application.
- *
- * @param {boolean} [options.test]          Boolean to determine if the application is running in test mode.
- * @param {boolean} [options.exitWhenDone]  Boolean to determine whether to automatically exit.
- */
-function Application (options) {
-  options = options || {};
-  this.pkgJson = require('../../package.json');
-  this.windows = [];
-
-  this.handleEvents();
-  this.openWithOptions(options);
-}
+import app              from 'app';
+import ipc              from 'ipc';
+import path             from 'path';
+import shell            from 'shell';
+import BrowserWindow    from 'browser-window';
+import { EventEmitter } from 'events';
+import _                from 'underscore-plus';
+import AppMenu          from './appmenu';
+import AppWindow        from './appwindow';
```

Comment:
Only one smallish thing left. Could you please remove the extra spaces to be in line with the repo style. Thank you